content/en/docs/reference/kubectl/_index.md

```diff
@@ -1,5 +1,551 @@
 ---
-title: "kubectl"
+title: Command line tool (kubectl)
```

Comment:
nit: Should the title contain Kubernetes?
Kubernetes command-line tool (kubectl)

Reply:
@kbhawkey let's save that for a different PR; that would be a change beyond the page move that this PR's title covers.